snowpack/src/commands/dev.ts

```diff
@@ -1376,7 +1412,12 @@ export async function startServer(commandOptions: CommandOptions): Promise<Snowp
     getServerRuntime: (options) => getServerRuntime(sp, options),
     async shutdown() {
       await watcher.close();
-      server.close();
+      await new Promise<void>((resolve, reject) => {
```

Comment:
Added this to help the test runner really knowing when everything is closed, and avoid hanging indefinitely when done... I must confess, this hasn't proven a massive success yet (the child process running the test currently has to forcefully exit to avoid hanging -- main suspect is Chokidar's FS watchers). Yet, I believe this is still cleaner to report the async completion, as is made in this change.